app/views/state_file/questions/return_status/_pending.html.erb

```diff
@@ -5,7 +5,7 @@
 </div>
 
 <h1 class="h2">
-  <%= @title %>
+  <%= t("state_file.questions.return_status.pending.title", state_name: States.name_for_key(params[:us_state].upcase)) %>
```

Comment:
I feel like we should be able to use the state_name method from the StateFileControllerConcern or some other place so it's a little less verbose than `States.name_for_key(params[:us_state].upcase`, but this is your call.

Reply:
I guess I'm wary to do this since `States.name_for_key` is already a part of a the helper `states.rb`, but I agree it is kind of verbose...